doc/source/ray-core/monitoring-debugging/troubleshoot-performance.rst

```diff
@@ -0,0 +1,57 @@
+Troubleshooting Performance
```

Comment:
All contents are just moved from original troubleshooting.rst. I will improve the contents in the follow-up PR